config/locales/en.yml

```diff
@@ -509,17 +497,22 @@ en:
               not_an_integer: The percentage that they own of the home must be a whole number
         housing_costs_form:
           attributes:
+            housing_benefit_relevant:
+              inclusion: Select yes if housing benefit is claimed at the home your client lives in
             housing_payments:
               blank: Enter the housing payments amount. Enter 0 if this does not apply.
               not_a_number: Housing payments must be a number. Enter 0 if this does not apply.
             housing_benefit_value:
-              blank: Enter the Housing Benefit amount. Enter 0 if this does not apply.
-              not_a_number: Housing Benefit must be a number. Enter 0 if this does not apply.
+              blank: Enter the Housing benefit amount.
+              not_a_number: Housing benefit must be a number.
+              greater_than: Housing benefit must be more than 0
+              greater_than_or_equal_to: Housing benefit must be must be 0 or more
               exceeds_costs: Housing benefit cannot be higher than housing costs
+              housing_benefit_legacy_presence: Enter the Housing benefit amount. Enter 0 if this does not apply.
             housing_payments_frequency:
-              blank: Select frequency of housing payments.
+              inclusion: Select frequency of housing payments.
             housing_benefit_frequency:
-              blank: Select frequency of Housing Benefit.
+              inclusion: Select frequency of Housing Benefit.
```

Comment:
should benefit be a lower case b to match elsewhere?

Reply:
Turns out Upper case was correct according to Laurence - have made consistent